clientv3/client.go

```diff
@@ -183,9 +218,21 @@ func newClient(cfg *Config) (*Client, error) {
 		c := credentials.NewTLS(cfg.TLS)
 		creds = &c
 	}
+
 	// use a temporary skeleton client to bootstrap first connection
 	ctx, cancel := context.WithCancel(context.TODO())
-	conn, err := cfg.RetryDialer(&Client{cfg: *cfg, creds: creds, ctx: ctx})
+	bootstrapClient := &Client{
```

Comment:
can we get rid of this bootstrap client? here we just need to get the initial conn, right?